File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.java
class Solution {
    public int solution(String s) {
        String[] str = s.split(" ");
        int answer = 0;
        int previous = 0;

        for (int i = 0; i < str.length; i++) {
            if (str[i].equals("Z")) {
                answer -= previous;
            } else {
                previous = Integer.parseInt(str[i]);
                answer += previous;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    var answer = 0;
    let str = s.split(" ");
    
    for(let i =0; i<str.length; i++){
        if(str[i]==="Z"){
         answer -= Number(str[i-1]);
        }else{
        answer += Number(str[i]);
        }   
    }
    
    return answer;
}